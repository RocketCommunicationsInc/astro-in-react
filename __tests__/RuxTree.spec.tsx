/**
 * @jest-environment jsdom
 */

test("use jsdom in this test file", () => {
  const element = document.createElement("div");
  expect(element).not.toBeNull();
});

import React from "react";
import { RuxTree, RuxTreeNode } from "../src";
import "@testing-library/jest-dom";
import renderer from "react-test-renderer";

describe("RuxTree", () => {
  it("should be rendered by react", () => {
    const container = renderer.create(
      <RuxTree>
        <RuxTreeNode>Kid</RuxTreeNode>
      </RuxTree>
    );
    let tree = container.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
