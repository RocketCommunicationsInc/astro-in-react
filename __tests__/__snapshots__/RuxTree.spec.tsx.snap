// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RuxTree should be rendered by react 1`] = `
<rux-tree>
  <rux-tree-node>
    Kid
  </rux-tree-node>
</rux-tree>
`;
